homeassistant/components/spotify/strings.json

```diff
@@ -10,6 +10,7 @@
     "abort": {
       "already_setup": "You can only configure one Spotify account.",
       "authorize_url_timeout": "Timeout generating authorize url.",
+      "no_url_available": "[%key::common::config_flow::abort::oauth2_no_url_available%]",
```

Comment:
This needs to be added to all integrations using the OAuth2 flow, not just Spotify